core/src/main/java/org/mule/runtime/core/internal/source/scheduler/SchedulerFlowProcessingTemplate.java

```diff
@@ -0,0 +1,106 @@
+import org.mule.runtime.core.internal.execution.SourceResultAdapter;
+import org.mule.runtime.core.privileged.event.BaseEventContext;
+import org.mule.runtime.extension.api.runtime.operation.Result;
+
+import java.util.List;
+import java.util.Map;
+
+import org.reactivestreams.Publisher;
+
+/**
+ * Custom scheduler's flow processing template.
+ *
+ * @since 4.3.0
+ */
+final class SchedulerFlowProcessingTemplate implements FlowProcessTemplate {
+
+  private final Processor messageProcessor;
+  private List<NotificationFunction> notificationFunctions;
+  private DefaultSchedulerMessageSource defaultSchedulerMessageSource;
+
+  SchedulerFlowProcessingTemplate(Processor messageProcessor,
+                                  List<NotificationFunction> notificationFunctions,
+                                  DefaultSchedulerMessageSource defaultSchedulerMessageSource) {
+    this.messageProcessor = messageProcessor;
+    this.notificationFunctions = notificationFunctions;
+    this.defaultSchedulerMessageSource = defaultSchedulerMessageSource;
+  }
+
+  @Override
+  public CheckedFunction<CoreEvent, Map<String, Object>> getSuccessfulExecutionResponseParametersFunction() {
+    return event -> emptyMap();
+  }
+
+  @Override
+  public CheckedFunction<CoreEvent, Map<String, Object>> getFailedExecutionResponseParametersFunction() {
+    return event -> emptyMap();
+  }
+
+  @Override
+  public SourceResultAdapter getSourceMessage() {
+    // payloadMediaTypeResolver is not needed since transmitted results is not a collection. Also, the cursorProviderFactory isn't
+    // needed either since no value is being communicated bu the ResultAdapter. This implies that no content is needed to be
+    // streamed.
+    return new SourceResultAdapter(Result.builder().build(), null, ANY, false,
+                                   empty(), null);
+  }
+
+  @Override
+  public List<NotificationFunction> getNotificationFunctions() {
+    return notificationFunctions;
+  }
+
+  @Override
+  public CoreEvent routeEvent(CoreEvent muleEvent) throws MuleException {
+    return messageProcessor.process(muleEvent);
+  }
+
+  @Override
+  public Publisher<CoreEvent> routeEventAsync(CoreEvent event) {
+    return just(event).transform(messageProcessor);
+  }
+
+  @Override
+  public Publisher<CoreEvent> routeEventAsync(Publisher<CoreEvent> eventPub) {
+    return from(eventPub).transform(messageProcessor);
+  }
+
+  @Override
+  public void sendResponseToClient(CoreEvent response, Map<String, Object> parameters, CompletableCallback<Void> callback) {
+    ((BaseEventContext) response.getContext()).success();
+  }
+
+  @Override
+  public void sendFailureResponseToClient(MessagingException exception,
+                                          Map<String, Object> parameters,
+                                          CompletableCallback<Void> callback) {
+    ((BaseEventContext) exception.getEvent().getContext()).error(exception);
```

Comment:
same as above

Reply:
Confirmed. It's configured in the processor chain error handling.